demos/interactive_face_detection_demo/detectors.cpp

```diff
@@ -613,10 +613,15 @@ std::string EmotionsDetection::operator[] (int idx) const {
     }
 
     auto emotionsValues = emotionsBlob->buffer().as<float *>();
-    auto outputIdxPos = emotionsValues + idx;
+    auto outputIdxPos = emotionsValues + idx*numOfChannels;
```

Comment:
Hi @zimka , 

Thanks a lot for the solution and your contribution.
Actually, we have already found it and fixed in the internal working tree. Moreover, this demo is completely redesigned in this internal release. So, next release will overwrite this part of source code and fix this problem.

But I can merge it anyway to have your solution working until our release is published.

Just one thing is missed. Unfortunately we don't have completely working CI for open source yet. And code style check was missed here. So, please put additional spaces around `*` symbol.

Reply:
Hi @vmatveic, it is a very minor bug of course, but just in my opinion it still is better to fix it, than not, until next release is available.

Reply:
Hi,

Actually it's not so minor because impacts a user by wrong results ;) Sure, I will merge, please just update it with required spaces.

Reply:
Fixed the spaces, please check it.